src/Admin/Views/Home/Index.cshtml

```diff
@@ -19,7 +19,7 @@
                 }
             });
 
-            fetch('@Url.Action("GetLatestDockerHubVersion", new { repository = "api" })').then((response) => {
+            fetch('@Url.Action("GetLatestVersion", new { repository = "core" })').then((response) => {
```

Comment:
With this change "repository" doesn't make much sense. What you think of calling it `project` instead?

Reply:
yes and i already changed it on the method signature, better to keep them equal.